osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -59,32 +62,30 @@ private void restartRequested()
         [BackgroundDependencyLoader]
         private void load()
         {
-            InternalChild = content = new Container
+            InternalChild = facadeContainer = CreateFacadeContainer();
```

Comment:
leave this called content. with it called `facadeContainer` it makes the rest of the class hard to read (you begin to question why transforms are being applied to it).